Eastern-Research-Group/mywaterway
# feature/154_popups-surrounding-waterbodies-flatten

## Related Issues:
* https://jira.epa.gov/browse/HMW-154

## Main Changes:
* Updated the surrounding waterbodies layer to allow popups on the on the protect and restore panels when users click inside of a HUC.

## Steps To Test:
1. Navigate to http://localhost:3000/community/allagash,%20me/protect
2. Click on a waterbody inside of the HUC, that is part of the surrounding waterbodies layer (i.e., not a blue waterbody)
3. Verify the popup works
4. Click on the restore tab
5. Repeat steps 2 and 3.

## TODO:
